analyse_performance.py

```
@@ -0,0 +1,490 @@
+COLUMN_WIDTH = 256.5 # in points, for two-column layout
+
+
+class Analyzer:
+    """
+    Class definition for the Analyzer. 
+    """
+
+    # Define the relevant file names which must be deleted 
+    FILE_TEMPLATES = {"walls": "walls.{}",
+                      "tflow": "tflow.{}",
+                      "tdat": "tdat.{}"}
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)